src/components/search/Search.jsx

```diff
@@ -40,6 +45,12 @@ const Search = () => {
     subscriptionLicense,
     offerCatalogs,
   });
+  const [pathwayUuid, setPathwayUuid] = useState(localStorage.getItem('pathwayUuid'));
```

Comment:
i have thought about this solution, although ideally we shouldnt be using this approach, but i think you can decide better if we need to alter the approach, or tackle this in tech debt or maybe move with this solution until we have a dedicated page for pathway instead of modal, in that case it will be very straight forward. 

you may want to discuss with someone but these are my 2 cents

Reply:
I have added a new page route for the pathway modal for now and discarded the local storage approach. Can you please review again? The new path would now look like `enterprise-name/search/pathway-uuid` instead of `enterprise-name/search/#pathway-uuid`